content/14.acknowledgements.md

```diff
@@ -11,4 +11,4 @@ All other authors have declared no competing interests.
 ## Acknowledgements {#acknowledgements}
 
 The authors would like the thank Daniel Himmelstein and the developers of Manubot for creating the software that enabled the collaborative composition of this manuscript.
-We would also like to thank Fábio Madeira ([0000-0001-8728-9449](https://orcid.org/0000-0001-8728-9449)), Victor Greiff ([0000-0003-2622-5032](https://orcid.org/0000-0003-2622-5032)), Shyam Saladi ([0000-0001-9701-3059](https://orcid.org/0000-0001-9701-3059)), Anshul Kundaje ([0000-0003-3084-2287](https://orcid.org/0000-0003-3084-2287)), Brett K. Beaulieu-Jones ([0000-0002-6700-1468](https://orcid.org/0000-0002-6700-1468)), Paul Brodersen ([0000-0001-5216-7863](https://orcid.org/0000-0001-5216-7863)), Michael M. Hoffman ([0000-0002-4517-1562](https://orcid.org/0000-0002-4517-1562)), and Isaac Lazzeri for their contributions to the discussions that comprised the initial stage of the drafting process.
+We would also like to thank Fábio Madeira ([0000-0001-8728-9449](https://orcid.org/0000-0001-8728-9449)), Victor Greiff ([0000-0003-2622-5032](https://orcid.org/0000-0003-2622-5032)), Shyam Saladi ([0000-0001-9701-3059](https://orcid.org/0000-0001-9701-3059)), Anshul Kundaje ([0000-0003-3084-2287](https://orcid.org/0000-0003-3084-2287)), Brett K. Beaulieu-Jones ([0000-0002-6700-1468](https://orcid.org/0000-0002-6700-1468)), Paul Brodersen ([0000-0001-5216-7863](https://orcid.org/0000-0001-5216-7863)), Michael M. Hoffman ([0000-0002-4517-1562](https://orcid.org/0000-0002-4517-1562)), and Isaac Lazzeri for their contributions to the discussions that comprised the initial stage of the drafting process. This work has been supported in part by the Biostatistics and Bioinformatics Shared Resource at the H. Lee Moffitt Cancer Center & Research Institute, an NCI designated Comprehensive Cancer Center (P30-CA076292).
```

Comment:
```suggestion
We would also like to thank Fábio Madeira ([0000-0001-8728-9449](https://orcid.org/0000-0001-8728-9449)), Victor Greiff ([0000-0003-2622-5032](https://orcid.org/0000-0003-2622-5032)), Shyam Saladi ([0000-0001-9701-3059](https://orcid.org/0000-0001-9701-3059)), Anshul Kundaje ([0000-0003-3084-2287](https://orcid.org/0000-0003-3084-2287)), Brett K. Beaulieu-Jones ([0000-0002-6700-1468](https://orcid.org/0000-0002-6700-1468)), Paul Brodersen ([0000-0001-5216-7863](https://orcid.org/0000-0001-5216-7863)), Michael M. Hoffman ([0000-0002-4517-1562](https://orcid.org/0000-0002-4517-1562)), and Isaac Lazzeri for their contributions to the discussions that comprised the initial stage of the drafting process.
```
The intended journal has a separate funding statement section, which we'll compile based on the funding information you provided in `content/metadata.yaml`.  We don't need to list this in the acknowledgements.

Reply:
It's a weird system, but my organization likes to track the shared resource usage through acknowledgements. We are encouraged to include the specific cores whenever possible. If it is not a big deal, I would like to please leave it in. If not, somewhere down the line I will get an email from an administrator with my chair CC'd asking me why I did not include it.

Reply:
Let's save @pstew the hassle of getting unpleasant emails and just leave it in.

Reply:
That's fine with me